src/gamercoder215/smpcore/utils/ItemsManager.java

```diff
@@ -0,0 +1,104 @@
+package com.stephen.plugin;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.bukkit.Material;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.attribute.AttributeModifier;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+public class ItemsManager {
+
+	public static ItemStack ThrowingKnife;
+	
+	public static ItemStack ThrowingKnife2;
+	
+	public static ItemStack ThrowingKnife3;
+	
+	public static ItemStack ThrowingKnife4;
+	
+	
+	
+	public static void init() {
```

Comment:
I would make these public, then convert void to ItemStack (return that ItemStack), then use the command to add that ItemStack to the inventory. Makes it a lot more simpler.

Reply:
Check ItemFetcher.java for more information.